src/irr/video/COpenGLSpecializedShader.cpp

```diff
@@ -272,6 +272,7 @@ void COpenGLSpecializedShader::setUniformsImitatingPushConstants(const uint8_t*
 auto COpenGLSpecializedShader::compile(const COpenGLPipelineLayout* _layout) const -> std::pair<GLuint, SProgramBinary>
 {
 	spirv_cross::CompilerGLSL comp(reinterpret_cast<const uint32_t*>(m_spirv->getPointer()), m_spirv->getSize()/4ull);
```

Comment:
can you cache this compiler?

Reply:
what did you end-up doing in the end?

Reply:
would u prefer it to be cached in COpenGLSPecShader or somehow in pipeline cache?

Reply:
pipeline cache, but in the second map/cache that's only for IGPUShader

Reply:
what happened in the end?

Reply:
COpenGLPipelineCache caches parsed spir-v and upon GL shader creation tries to find parsed spir-v to use it for faster spirv-cross compiler construction (in order to specialize, reorder bindings and generate final GLSL)